nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/custom/Assign.java

```diff
@@ -46,6 +47,10 @@ public Assign(INDArray[] inputs, INDArray[] outputs) {
         super(null,inputs, outputs);
     }
 
+    public Assign(INDArray x, INDArray y ) {
+        this( new INDArray[]{y ,x},new INDArray[]{y}); // TODO: Still check. y cannot be null, must be same shape as x.
+    }
```

Comment:
This should be:
```
    public Assign(@NonNull INDArray x, INDArray z ) {
        this( new INDArray[]{x}, wrapOrNull(z));
    }
```
Where wrapOrNull is an inherited method (already exists)

Reply:
assign has been deleted.